addons/mail/static/src/core/public_web/messaging_menu.scss

```diff
@@ -2,6 +2,7 @@
     width: 475px;
     min-height: 50px;
     max-height: Min(calc(#{ $o-dropdown-max-height } - 5px), 630px); // -5px for borders
+    z-index: 1000;
```

Comment:
`MessagingMenu` using `z-index: 1000` from dropdown is obsolete.
Since the comment, messaging menu is in popover, and the z-index of popover is `1070` so there's room